java/ql/src/semmle/code/java/security/UnsafeDeserialization.qll

```diff
@@ -50,6 +55,29 @@ class SafeKryo extends DataFlow2::Configuration {
   }
 }
 
+class SafeJsonIo extends DataFlow2::Configuration {
```

Comment:
Should be moved to the JsonIO.qll lib

Reply:
Haha, I see other tracing flows are in this `qll` file, so put it here.